test/extensions/filters/http/ext_proc/ext_proc_integration_test.cc

```diff
@@ -202,6 +204,19 @@ class ExtProcIntegrationTest : public HttpIntegrationTest,
       setUpstreamProtocol(Http::CodecType::HTTP2);
       setDownstreamProtocol(Http::CodecType::HTTP2);
     }
+
+    // Add tracing filter to the config.
+    if (config_option.add_tracing) {
+      config_helper_.addConfigModifier([&](HttpConnectionManager& cm) {
+        envoy::config::trace::v3::OpenTelemetryConfig provider_config;
```

Comment:
It is not recommended to have cross-filter dependence as it could be disable in people's custom build.

Would it possible to have a simple test filter similar to OpenTelemetryConfig?

https://github.com/envoyproxy/envoy/tree/main/test/integration/filters this folder contains test filters for your reference. 

From a quick scan, OpenTelemetryConfig performs http/gRPC callout. `server_factory_context_filter_config.proto` could be a reference, though it is only gRPC based callout.  
Besides, I am wondering if your feature can be validated by local filter/extension (i.e., no callout) , which might also make your test easier.